ansibullbot/utils/extractors.py

```diff
@@ -493,12 +494,13 @@ def metadata(self):
             self._METADATA = self.get_module_metadata()
         return self._METADATA
 
-    def get_module_authors(self):
-        """Grep the authors out of the module docstrings"""
+    @property
+    def docs(self):
+        if self._DOCSTRING is not None:
+            return self._DOCSTRING
 
         documentation = u''
         inphase = False
```

Comment:
plz return the removed line here:
```suggestion
         inphase = False

```